tests/core/daemon/test_daemon.py

```diff
@@ -59,6 +59,13 @@ async def is_running(self, request: Dict[str, Any]) -> Dict[str, Any]:
 }
 
 
+def add_private_key_response_data(fingerprint: str) -> Dict[str, object]:
```

Comment:
```suggestion
def add_private_key_response_data(fingerprint: int) -> Dict[str, object]:
```

`mypy` just doesn't complain yet because we have `test_daemon.py` excluded in the `mypy.ini`.